components/drawer-layout/DrawerMenuContent.tsx

```
@@ -20,62 +20,71 @@ import { Platform, ScrollView } from "react-native";
 import useBreakpoints from "@/hooks/use-breakpoints";
 import { DrawerActions } from "@react-navigation/native";
 import { useNavigation, useRouter } from "expo-router";
+import OrganizationSwitcherMenu from "../org-switcher";
+import { useOrganizationContext } from "@/contexts";
 
 interface DrawerMenuContentProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)